app/components/account/details/account-details.html

```diff
@@ -16,19 +16,81 @@
             </md-card-actions>
         </md-card>
         <div class="account-links">
-            <md-card class="link" routerLink="/jobs" (click)="selectAccount(account.id)">
-                <i class="fa fa-tasks fa-lg"></i>
-                <div>Jobs</div>
+            <md-card class="link">
+                <md-card-header>
+                    <md-card-title><h2>Job status</h2></md-card-title>
+                </md-card-header>
+                <md-card-content class="account-card-content">
+                    <div>
+                        <bl-loading [status]="jobData.status | async">
+                            <p *ngIf="(jobData.items | async).size === 0 && !loading"
```

Comment:
There should be a `bl-no-item` component you give the size and the status and it will know when to show the message